tests/models/test_dag.py

```diff
@@ -2102,6 +2102,11 @@ def test_schedule_dag_param(self, kwargs):
 
 
 class TestDagModel:
+    def setup_method(self):
+        clear_db_dags()
+        clear_db_datasets()
+        clear_db_runs()
```

Comment:
```suggestion
    def _clean(self):
        clear_db_dags()
        clear_db_datasets()
        clear_db_runs()

    def setup_method(self):
        self._clean()

    def teardown_method(self):
        self._clean()
```

Probably worth running this on both setup and teardown.